WordPress/Classes/ViewRelated/Reader/ReaderDetailViewController.swift

```diff
@@ -497,16 +481,13 @@ public class ReaderDetailViewController : UIViewController, UIViewControllerRest
 
     }
 
+
     private func configureRichText() {
-        richTextView.delegate = self
-        let fontSize = WPStyleGuide.Detail.contentFontSize
-        let lineHeight = WPStyleGuide.Detail.contentLineHeight
-        var textOptions = richTextView.textOptions
-        textOptions[DTDefaultFontSize] = WPStyleGuide.Cards.contentFontSize
-        textOptions[DTDefaultLineHeightMultiplier] = lineHeight / fontSize
```

Comment:
![Kill it with fire](https://cloud.githubusercontent.com/assets/8739/20061666/0de41ace-a501-11e6-94bb-eee67ac127b7.gif)